server/sonar-server/src/main/java/org/sonar/server/project/ws/CreateAction.java

```diff
@@ -116,13 +116,15 @@ private CreateWsResponse doHandle(CreateRequest request) {
       OrganizationDto organization = support.getOrganization(dbSession, ofNullable(request.getOrganization())
         .orElseGet(defaultOrganizationProvider.get()::getKey));
       userSession.checkPermission(PROVISION_PROJECTS, organization);
+      String visibility = request.getVisibility();
+      Boolean changeToPrivate = visibility == null ? dbClient.organizationDao().getNewProjectPrivate(dbSession, organization) : "private".equals(visibility);
```

Comment:
remark: I don't understand what `getNewProjectPrivate` does from the name and it doesn't follow our name conventions.
I know you didn't create the method